modules/virt-about-creating-network-bridges.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc
+
+:_content-type: CONCEPT
+[id="virt-about-creating-network-bridges_{context}"]
+= About creating network bridges
+
+You can attach virtual machines to multiple networks by using Linux bridges.
```

Comment:
I'm not sure I like starting this section with "You can". It makes it sound like an option. Suggestion: "You attach virtual machines ..."

Reply:
Personally I prefer directives and I usually get push back from peer reviewers when I don't say "can". So, I agree.